monitoring/docker-compose.yml

```
@@ -0,0 +1,60 @@
+version: '3'
+
+services:
+  indexer:
+    container_name: "indexer"
+    build:
+      context: ./../
+      dockerfile: "./monitoring/Dockerfile-indexer"
+    environment:
+      PORT: 8888
+      CONNECTION_STRING: ${DB_CONNECTION}
```

Not sure we need the `.env` file for the default, because the DB credentials are defined in this file:
```suggestion
      CONNECTION_STRING: "${DB_CONNECTION:-host=indexer-db port=5432 user=algorand password=algorand dbname=indexer_db sslmode=disable}"```